### ISSUE TITLE
Add `OmitOf` and `PickOf` to modify object predicate function

### ISSUE THREAD

**Issue by lambdalisue** [lambdalisue/deno-unknownutil#52]
To create subpredicate functions from an existing one. For example, assume you can use the following code:

```ts
const isArticle = is.ObjectOf({
  id: is.String,
  subject: is.String,
  body: is.String,
});

const isArticleCreateParams = is.OmitOf(isArticle, ["id"]);
// Equivalent to
//const isArticleCreateParams = is.ObjectOf({
//  subject: is.String,
//  body: is.String,
//});
```

I'm uncertain about whether `OmitOf` and `PickOf` make sense. I'd like to maintain the `*Of` convention, but if it seems unclear, perhaps we should consider naming them `OmitFrom` and `PickFrom` instead, or explore other alternatives.

**Comment by lambdalisue** [comment]
I'm working on it.

**Comment by lambdalisue** [comment]
Closed by #57
